server/execution/api/manvmap.py

```diff
@@ -0,0 +1,49 @@
+from random import random
+
+from flask import Blueprint
+from flask_jwt_extended import jwt_required
+
+from execution.utils import util
+
+api = Blueprint("api-manvmap", __name__)
+
+
+@api.get("/mapdata")
+@jwt_required()
+def get_map_data():
+    """ Returns mock map data. """
+    try:
+        execution, _ = util.get_execution_and_player()
+        return {
+            "patient_positions": [
+                {"position": {"x": random() * 1000, "y": random() * 1000},
+                 "patient_id": patient.id,
+                 "classification": patient.classification.name}
+                for patient in execution.scenario.patients.values()
+            ],
+            "location_positions": [
+                {"position": {"x": random() * 1500, "y": random() * 1500},
```

Comment:
Out of interest: is it possible for a patient to be placed inside a building? As far as i know buildings are displayed as blocked space?

Reply:
Yes indeed, but thats okay for now as the server side is out of the scope for my individual part.  
It is assumed that the patients are not in buildings, but as the number of patients is generally not known i generate the positions randomly.  
This might change as I might provide a few positions which are not in the buildings.  
But perspectively the admin providing the map data would be responsible for sensible positions.